doc-Installing_on_Microsoft_Azure/topics/installation.adoc

```diff
@@ -157,15 +254,143 @@ New-AzureRmVM -ResourceGroupName $ResourceGroupName -Location $Location -VM $Vir
 +
 
 
-[[uploading-the-appliance-using-azure-command-line-interface]]
-==== Uploading the Appliance Using Azure Command-Line Interface
+[NOTE]
+====
+These are the procedural steps as of the time of writing. For more information, see the following Azure documentation. 
 
-Although it is recommended to use the PowerShell method, you can upload the appliance to an Azure environment using the CLI method. See https://azure.microsoft.com/en-us/documentation/articles/xplat-cli-azure-resource-manager/ for steps on how to create and manage Azure resources using the Azure Command-Line Interface (Azure CLI).
+* https://azure.microsoft.com/en-us/documentation/articles/powershell-azure-resource-manager
 
+The steps covered in the following article are for a Windows machine, however, most of the items are common between Windows and Linux.
 
+* https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-create-powershell
+====
+
+
+[[uploading-the-appliance-using-azure-cli]]
+==== Uploading and Provisioning the {product-title_short} Virtual Appliance Using Azure Command-Line Interface
+
+You can upload the appliance to an Azure environment using the Azure Command-Line Interface (Azure CLI) following the process below.
+
+
+.Install Azure CLI 2.0:
+
+Complete the steps below to install Azure CLI 2.0 using curl. See link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli[] for other installation methods.
+
+. Make sure Python is updated and install the prerequisite packages.
++
+----
+$ sudo yum update
+$ sudo yum install -y gcc libffi-devel python-devel openssl-devel
+----
++
+. Install Azure CLI 2.0.
++
+----
+$ curl -L https://aka.ms/InstallAzureCli | bash
+----
++
+
+
+
+
+.Upload and Provision the {product-title_short} Virtual Appliance Using Azure CLI:
+
+Complete the following steps to upload and provision the {product-title} virtual appliance you converted to a fixed-size VHD format and properly aligned per requirement using the procedure in xref:converting-aligning-cfme-appliance-image[].
+
+
+. Upload the image to the storage container. It may take several minutes. Note: Enter `az storage container list` to get the list of storage containers.
++
+----
+$ az storage blob upload --account-name <storage-account-name> --container-name <container-name> --type page --file <path-to-vhd> --name <image-name>.vhd
+
+Example:
+
+$ az storage blob upload --account-name azrhelclistact --container-name azrhelclistcont --type page --file cfme-azure-5.7.2.1-1.x86_64.vhd --name cfme-azure-5.7.2.1-1.x86_64.vhd 
+Percent complete: %100.0
+----
++
+. Get the URL for the uploaded .vhd file. You will need to use this URL in the following step.
```

Comment:
You get the URL for the .vhd file as an output of this step which will be used in the next step. I've tweaked the step a bit to make it clear. Hope that makes sense.